frb_codegen/src/generator/dart/ty_enum.rs

```diff
@@ -55,8 +93,7 @@ impl TypeDartGeneratorTrait for TypeEnumRefGenerator<'_> {
                         )
                     }
                 })
-                .collect::<Vec<_>>()
-                .join("\n"),
+                .join("else\n"),
```

Comment:
Not sure what is happening here... using "else" to join multiple strings?

Reply:
Shortcut for if-else branches, not really essential.

Reply:
I see. Not sure this makes code cleaner or more confusing, since usually "join" is for multiple (i.e. possibly >=3) items